fhir/resources/DSTU2/allergyintolerance.py

```diff
@@ -23,7 +23,7 @@ class AllergyIntolerance(DomainResource):
 
     resource_type = Field("AllergyIntolerance", const=True)
 
-    category: ListType[fhirtypes.Code] = Field(
+    category: fhirtypes.Code = Field(
         None,
         alias="category",
         title="List of `Code` items (represented as `dict` in JSON)",
```

Comment:
Please change //List of `Code` items (represented as `dict` in JSON)// to // Type ´Code´.